### PR TITLE
Add facility grades UI

### PR DESCRIPTION
**Pull request by timcheadle**
**Story card:** [ch2059](https://app.clubhouse.io/simpledotorg/story/2059/add-facility-grades-based-on-performance-score)

## Because

CVHOs want to see facilities grouped by performance grades, simple letter grades A, B, C, and D.

## This addresses

This adds letter grades based on the `PerformanceScore` and displays them (instead of rank) in the UI.

image

### CODE REVIEW

File: app/views/my_facilities/ranked_facilities/show.html.erb [hunk 1]
```diff
@@ -64,13 +61,12 @@
           </tr>
         </thead>
         <tbody>
-          <% @facilities.each_with_index do |facility, rank| %>
+          <% @facilities.each_with_index do |facility| %>
```
Comment (harimohanraj89) [inline app/views/my_facilities/ranked_facilities/show.html.erb:64]: We don't need `with_index` anymore.

Comment (harimohanraj89) [inline app/views/my_facilities/ranked_facilities/show.html.erb:64]: We don't need `with_index` anymore.

File: app/controllers/my_facilities/ranked_facilities_controller.rb [hunk 2]
```diff
@@ -37,7 +37,7 @@ def show
     end
 
     # Sort facilities by overall score, highest to lowest
```
Comment (harimohanraj89) [inline app/controllers/my_facilities/ranked_facilities_controller.rb:39]: We can update (or even remove) this comment now.

**Review by harimohanraj89 — APPROVED** [review]
A few non-blocking suggestions.

I'm guessing the updates to the scoring algorithm itself will come later? This just divies up the exisiting algorithm into letter grades?

**Comment by timcheadle** [comment]
@harimohanraj89 the scoring changes already merged; see #1590

**Pull request merged**